rpc_util.go

```diff
@@ -541,6 +541,9 @@ type parser struct {
 	// The header of a gRPC message. Find more detail at
 	// https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md
 	header [5]byte
+
+	// useBytesPool indicates whether to use bytes pool to allocate
+	useBytesPool bool
```

Comment:
This could also be renamed `useSharedBuffers` or something of that sort.

Reply:
Same as above.